dashboard/app/controllers/teacher_feedbacks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class TeacherFeedbacksController < ApplicationController
```

Comment:
This is an API controller, right? A REST API that our React client will call via AJAX? If so, it should go under `/api/v1/`